runelite-client/src/main/java/net/runelite/client/config/FontType.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2018, Cameron <https://github.com/noremac201>
```

Comment:
Maybe fix your copyright, unless this is you

Reply:
What do you mean by this. Can you give an example of what you are referring to?